### PR TITLE
Plugin update

### PR DESCRIPTION
**Pull request by scottnath**
Bringing boolean up to date with our process and requirements

---

`DCO 1.1 Signed-off-by: Scott Nath <github@scottnath.com>`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
+  - npm run coverage
+  - npm run semantic-release
+branches:
+  except:
+    - /^v\d+\.\d+\.\d+$/
+#########################
+## Notifications
+#########################
 notifications:
-  slack:
```
Comment (Snugug) [inline .travis.yml:8]: Where did Slack notifications go?

**Comment by Snugug** [comment]
@scottnath Just the slack notifications going away needs to be fixed

**Comment by scottnath** [comment]
@Snugug slack added

File: .travis.yml [hunk 2]
```diff
@@ -6,15 +6,14 @@ cache:
   directories:
   - node_modules
 before_script:
-  - npm prune
+- npm prune
```
Comment (Snugug) [inline .travis.yml]: Check your indentation

**Pull request merged**